features/rum/src/appleTest/kotlin/com/datadog/kmp/rum/internal/RumMonitorAdapterTest.kt

```diff
@@ -676,6 +735,15 @@ class RumMonitorAdapterTest {
             }
         }
 
+    private val FailureReason.native: DDRUMFeatureOperationFailureReason
```

Comment:
maybe it's make sense to put such code into to some unified space to use them for both production and test code? seems that it repeated several times(Some examples are similar but for different platforms, but still).